README.md

```diff
@@ -59,12 +59,14 @@ You can also [build the client yourself](#building-the-client). While the client
 
 The plan for delivering next Windows Package Manager releases is described included in our [discussions](https://github.com/microsoft/winget-cli/discussions/2063), and will be updated as the project proceeds.
 
-## Overview of the  Windows Package Manager
+## Overview of the Windows Package Manager
 
 The **Windows Package Manager** is a tool designed to help you quickly and easily discover and install those packages that make your PC environment special.  By using the **Windows Package Manager**, from one command, you can install your favorite packages:
 
 `winget install <package>`
 
+Note: the default install location for packages is `%LOCALAPPDATA%\Microsoft\WinGet\Packages`
```

Comment:
This is not accurate. That is only the default install location for Portable Packages installed in the user scope.
If you run `winget --info` on Winget 1.5 or newer, you will see this is just one of three default directories for portable packages - and these are also configurable through `winget settings`
```
Winget Directories
-------------------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages
```

Additionally, applications which are not portable applications may have their own default install location which is set by the publisher and which WinGet has no control over.

Reply:
Thanks Trenly,
that is good to know, i thought there was only one directory because i had trouble finding the directory after running `winget install deno`

all good, will close off this PR as the info command gives a lot of info